mmv1/products/apphub/Workload.yaml

```diff
@@ -0,0 +1,241 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+base_url: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads
+create_url: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads?workloadId={{workload_id}}
+self_link: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads/{{workload_id}}
+id_format: projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads/{{workload_id}}
+import_format:
+  - projects/{{project}}/locations/{{location}}/applications/{{applications_id}}/workloads/{{workload_id}}
+name: Workload
+description: 'Workload represents a binary deployment (such as Managed Instance Groups (MIGs), GKE deployments, etc.) that performs the smallest logical subset of business functionality.
+  It registers identified workload to the Application.'
+autogen_async: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  constants: 'templates/terraform/constants/apphub_discovered_resource.go.erb'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: "workload_basic"
+    pull_external: true
+    primary_resource_id: "example"
+    config_path: "templates/terraform/examples/apphub_workload_basic.tf.erb"
+    vars:
+      application_id: "example-application-1"
+      service_project_attachment_id: "project-1"
+      ilb_network: "l7-ilb-network"
+      ilb_subnet: "l7-ilb-subnet"
+      instance_template: "l7-ilb-mig-template"
+      mig: "l7-ilb-mig1"
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+  - !ruby/object:Provider::Terraform::Examples
+    name: "workload_full"
+    pull_external: true
+    primary_resource_id: "example"
+    config_path: "templates/terraform/examples/apphub_workload_full.tf.erb"
+    vars:
+      application_id: "example-application-1"
+      service_project_attachment_id: "project-1"
+      display_name: "Example Service Full"
+      description: "Register service for testing"
+      business_name: "Alice"
+      business_email: "alice@google.com"
+      developer_name: "Bob"
+      developer_email: "bob@google.com"
+      operator_name: "Charlie"
+      operator_email: "charlie@google.com"
+      ilb_network: "l7-ilb-network"
+      ilb_subnet: "l7-ilb-subnet"
+      instance_template: "l7-ilb-mig-template"
+      mig: "l7-ilb-mig1"
+    test_env_vars:
+      org_id: :ORG_ID
+      billing_account: :BILLING_ACCT
+properties:
+  - !ruby/object:Api::Type::String
+    name: name
+    output: true
+    description: "Identifier. The resource name of the Workload. Format:\n\"projects/{host-project-id}/locations/{location}/applications/{application-id}/workloads/{workload-id}\" "
+  - !ruby/object:Api::Type::String
+    name: displayName
+    description: 'Optional. User-defined name for the Workload. '
```

Comment:
You don't need to include `Optional` or `Required` in any of these descriptions; the generator will add that for you.